charts/home-assistant/README.md

```diff
@@ -155,6 +156,7 @@ The following tables lists the configurable parameters of the Home Assistant cha
 | `appdaemon.image.pullPolicy`                    | Image pull policy                                                                                                                                                                                                                         | `IfNotPresent`                      |
 | `appdaemon.haToken`                             | Home Assistant API token - you need to generate it in your Home Assistant profile and then copy here                                                                                                                                      | ``                                  |
 | `appdaemon.extraEnv`                            | Extra ENV vars to pass to the AppDaemon container                                                                                                                                                                                         | `{}`                                |
+| `appdaemon.configSubPath`                       | An optional subPath for the AppDaemon container's config volume mount   | `appdaemon`                          |
```

Comment:
This should probably also have an entry in the `values.yaml`.  Basically, if we make reference to a helm configuration setting in the readme or actual templates, we need to have a corresponding reference in the `values.yaml` (even if it's commented out)

Reply:
Done